pkg/apiserver/apiserver.go

```diff
@@ -67,12 +70,15 @@ func (c CompletedConfig) New() (*ServiceCatalogAPIServer, error) {
 	// giving it v1alpha1 version
 	apiGroupInfo.GroupMeta.GroupVersion = v1alpha1.SchemeGroupVersion
 
-	// TODO make storage work
-	//v1alpha1storage := map[string]rest.Storage{}
-	//
-	//v1alpha1storage["servicecatalog"] = apiservice.NewREST(c.RESTOptionsGetter.NewFor(apiregistration.Resource("servicecatalog")))
+	v1Alpha1Storage := map[string]rest.Storage{
```

Comment:
can you explain how each of these keys work? I don't understand what the result of this is.